### DISCUSSION TITLE
gamecube luigis mansion r2 Problem.

### DISCUSSION THREAD

**Discussion by jackmiller1986** [darthcloud/BlueRetro#325] [Q&A]
Hello Guys,
At first i love this Project. 😉

I have install an esp32 in my gamecube , and have Flash the newest Firmware 1.6.3.

I have testet with 3 Controllers, an ps5 , a ps3 , and a joycon remote.

The ps5 , and ps3 are nice with Connections. When i Connect two joycons, one of them lost permanently Connecticon.

But the most expensive Thing is, i have testet 3 title, Luigis mansion, mario Sunshine, and zelda windwaker.

Mario and windwaker works on both controller All Buttons inkl. The r2 Button. But only on luigis mansion , IT wont work , on ps5 and ps3 controller , its the only game , who not recocnize the r2 Button, But all other buttons works in lugis mansion without the r2 thats the only who not works . Is that a Firmware Bug? Because all other games works like a charm . Can anybody confirm that? And why break one joycon the Transmission?

I hope any body can help.

**Comment by darthcloud** [comment]
Already answer in #324.

Please don't double post.

**Comment by darthcloud** [comment]
@jackmiller1986 don't open new issue I already answer in #324

Did you go in the [presets config page](https://blueretro.io/blueretro_presets.html) and set the "GameCube Merge analog & digital trigger"

**Comment by jackmiller1986** [comment]
yes i do, the magic is, when i go to mario sunshine all buttons works fine, in zelda and other games too.

I have change multi fw from here, nothing happen in luigis mansion.

how can it be that, this game not response r2? But the oter games, this show me , that i must, the ESP32 install correctly, when al function works on other games. all 4 connections work too on the gamecube.

what can it be ??

**Reply by jackmiller1986** [reply]
I resolder all today , same Thing

**Comment by darthcloud** [comment]
I don't have Luigi's mansion, I ordered one and will test once received.

**Reply by jackmiller1986** [reply]
Thx allot my Friend my childs love to play this game . Can IT be a Software Solution? And can it be fix in the configs then ? Or can it be an esp32 Problem? But i wait and will See,  thx thx thx for Your awesome great work .

**Reply by jackmiller1986** [reply]
Any solution my friend ?

**Comment by darthcloud** [comment]
Fixed in v1.7.1

**Comment by jackmiller1986** [comment]
IT works perfectly , my Kids are So happy , thx you my friend , testet , with ps3/ps4/ps5 controller . Can closed.
